Makefile

```diff
@@ -1,137 +1,9 @@
-PYTHON=`which python3`
-PY2DSC=`which py2dsc`
+USE_VERSION_FILE:=TRUE
 
-PY2DSC_PARAMS?=--with-python2=false --with-python3=true
+# Needed here due to gitignore being generated in standalone.mk but Dockerfile.multi being added in docker.mk
+EXTRA_GITIGNORE_LINES+=Dockerfile.multi
```

Comment:
This is a bit odd because I think it worked before for standalone libraries. Do you know what changed to break it?

Reply:
Good catch. This was copy-paste from elsewhere so probably isn't needed there either!